### ISSUE TITLE
CDN Service

### ISSUE THREAD

**Issue by ytliuSVN** [hummingbird-dev/hummingbird-treeview#7]
Thanks for the plugin! Is there a CDN for Hummingbird Treeview?

**Comment by hummingbird-dev** [comment]
No, unfortunately not, sorry. Thanks for using the hummingbird-treeview. Enjoy.

**Comment by alexgivi** [comment]
@ytliuSVN  you can use such urls to files in this repo:
https://raw.githubusercontent.com/hummingbird-dev/hummingbird-treeview/master/hummingbird-treeview.js
https://raw.githubusercontent.com/hummingbird-dev/hummingbird-treeview/master/hummingbird-treeview.css
I don't know about speed, but you shouldn't download and add files to your project anyway.

**Comment by alexgivi** [comment]
hmm, it looks like i've made a mistake... github sends files as 'text/plain' and so it's a problem, browser doesn't link this files.
